### ISSUE TITLE
[Core] NDB_Page:createCheckbox() - Unused and undocumented param $closer

### ISSUE THREAD

**Issue by laemtl** [aces/Loris#6504]
**master branch**
 [php/libraries/NDB_Page.class.inc:createCheckbox()](https://github.com/aces/Loris/blob/master/php/libraries/NDB_Page.class.inc#L590)
Check and remove the below param if it's not necessary.

```
/**
     * Creates an HTML checkbox but does not add it to the form.
     *
     * FIXME: $closer is never used or explained. - Alexandra Livadas
     *
     * @param string $field   The fieldname for this checkbox
     * @param string $label   The label to attach to this checkbox
     * @param string $options Options to pass to HTML_QuickForm
     * @param string $closer  ?????
     *
     * @return array representing checkbox
     */
    function createCheckbox($field, $label, $options=null, $closer='</label>')
    {
        return $this->form->createElement(
            "advcheckbox",
            $field,
            $label,
            $options,
            array()
        );
    }
```
See #5687 for more details

**Comment by lingma** [comment]
Processed in PR #6396, not necessary now.

**Comment by lingma** [comment]
By the way, 23.0-release branch might be more recent than the master branch for the moment. @laemtl

**Comment by laemtl** [comment]
Thanks lingma, I see that this has been solved in 23.0-release, but the comment has not been removed.

**Comment by lingma** [comment]
Oh, sorry. I just removed from PR #6444 pending review, which is blocked by another PR.
